### PR TITLE
[Fixture] Togglable default locale loading

### LINKED ISSUE #10307: Fixtures: Avoid using `$baseLocale` at `LocaleFixture`
**Sylius version affected**: any

**Description**  
`LocaleFixture` adding `$baseFixture` to list of fixtures to load.
This not allow us to load `locale:` fixtures second time:

```bash
An exception occurred while executing 'INSERT INTO sylius_locale (code, created_at, updated_at) VALUES (?, ?, ?)' with params ["en_US", "2019-04-12 18:10:00", "2019-04-12 18:10:00"]:  
                                                                                                                                                                                          
  SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'en_US' for key 'UNIQ_7BA1286477153098'         
```

**Steps to reproduce**  
- Load default fixtures suite
- Load another suite with `orm_purger: false` and `locale:` fixture

**Possible Solution**  
- Avoid using `$baseLocale`. 
- Specify basic locale right at `fixtures.yml`:

```yaml
sylius_fixtures:
    suites:
        default:
            listeners:
                orm_purger: ~
                logger: ~
            fixtures:
                locale:
                    options:
                        locales:
                            - "%locale%"
```

I will make PR if you guys OK with this.
Let me know please if I can do this on `1.4` or even `1.3` rather than `master` - I'm not sure how strict BC compatibility should be...

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | yes
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes #10307
| License         | MIT

Opened to master as it is a new feature at the end.

### CODE REVIEW

**Comment by igormukhingmailcom** [comment]
Hi, guys. 
Any chance this PR will be merged?

**Comment by lchrusciel** [comment]
Hello Igor, 

this PR was red due to the problem with the side library. I have restarted a build.

File: src/Sylius/Bundle/CoreBundle/Fixture/LocaleFixture.php [hunk 1]
```diff
@@ -74,6 +78,9 @@ protected function configureOptionsNode(ArrayNodeDefinition $optionsNode): void
     {
         $optionsNode
             ->children()
+                ->scalarNode('load_default_locale')
+                    ->defaultTrue()
+                    ->end()
```
Comment (GSadee) [inline src/Sylius/Bundle/CoreBundle/Fixture/LocaleFixture.php]:
```
$optionsNode
    ->children()
        ->scalarNode('load_default_locale')->defaultTrue()->end()
        ->arrayNode('locales')->scalarPrototype()
;
``` 
?

**Comment by bartoszpietrzak1994** [comment]
Could you document the behavior that you want to add in this PR? It would be great if our docs informed the users about the `load_default_locale` flag.

**Comment by lchrusciel** [comment]
Good idea, but I don't know where should it be placed. We don't have any option documented around fixtures configuration.

Thus, we can open an issue about missing documentation after merging this PR

**Comment by Zales0123** [comment]
I'm a little bit afraid it can be written and forgotten :( Maybe we can add a quick note in [this chapter](https://docs.sylius.com/en/1.4/book/architecture/fixtures.html#what-data-is-loaded-by-fixtures-in-sylius) of documentation? Just to not forget it and **then** open an issue about missing documentation 😄

**Comment by lchrusciel** [comment]
I'm voting for merging this feature and let's focus on documentation form in separate PR (#10393)

**Comment by Zales0123** [comment]
Thank you, Łukasz! :1st_place_medal:

**Comment by igormukhingmailcom** [comment]
Thank you guys :)

**Pull request merged**